TableView/Protocols/ControlManagerProtocol.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ControlManagerProtocol.swift
+//  TableView
+//
+//  Created by Oleg Yankiwskyi on 4/26/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import Foundation
+
+protocol ControlManagerProtocol {
+    weak var delegeteFakeData: FakeDataProtocol? { get set }
+    func createMenu() -> [TypeItem]
+    func setDelegate(delegeteFakeData: FakeDataProtocol) 
```

Comment:
`setDelegate` seems to be excessive. `weak var delegeteFakeData` is enough, I think